lightly/api/api_workflow_upload_dataset.py

```diff
@@ -97,6 +111,10 @@ def lambda_(i):
                 return True
 
             filepath = dataset.get_filepath_from_filename(filename, image)
+
+            # get custom metadata (evaluates to None if there is none)
+            custom_metadata_ = filename_to_metadata.get(filename, None)
```

Comment:
Can we use another variable name here? I think we should not append `_`. Somehow it feels wrong xD
I might be wrong here. But if it's a private variable use `_custom_metadata`. Why can't we use `custom_metadata`?

Reply:
You're right, trailing underscores are only for conflicts with built-in names, from PEP 8:
> "`single_trailing_underscore_`: used by convention to avoid conflicts with Python keyword, e.g."